scripts/benchmarking.py

```diff
@@ -219,6 +219,7 @@ def recall(predicted, relevant, eval_at):
             console.print(f'Storage Backend {backend} failed: {e}')
 
 df = pd.DataFrame(find_by_vector_values)
+print(df)
```

Comment:
I guess you could add `df.to_csv('benchmark_results.csv')` so that results are stored and can easily be shared/loaded/printed later.